.gitignore

```diff
@@ -4,3 +4,4 @@ composer.lock
 composer.phar
 .php_cs
 .php_cs.cache
+.idea
```

Comment:
IMHO this ignore should be set on your GIT's global configuration.

Reply:
thanks, done!